src/data-loaders/defineQueryLoader.spec.ts

```diff
@@ -30,6 +31,6308 @@ describe.skip('defineQueryLoader', () => {
       // plugins: ({ pinia }) => [pinia],
     }
   )
+
+  describe("defineQueryLoader edge cases and error handling", () => {
```

Comment:
@coderabbitai remove the changes to this file. It's outdated and I will remove it myself